internal/pkg/template/template.go

```diff
@@ -224,8 +224,12 @@ func (t *Template) ProjectsFilePath() string {
 func (t *Template) TestsDir(ctx context.Context) (filesystem.Fs, error) {
 	if t.testsDir == nil {
 		if !t.fs.IsDir(ctx, TestsDirectory) {
-			return nil, errors.Errorf(`directory "%s" not found`, TestsDirectory)
+			err := t.fs.Mkdir(ctx, TestsDirectory)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
The folder 'test' wasn't created. The missing files (.keboola/manifest.json, main/meta.json...) must be stored in the tests folder. 

<img width="233" alt="image" src="https://github.com/user-attachments/assets/29a48c83-eff9-461a-a0e1-3cfcb3ef2123">